js/modules/swiperCommentsFunctionality.js

```diff
@@ -0,0 +1,122 @@
+import { setSlidesState } from "./swiperHelpers.js";
+import { switchSlidesStateHandlerOn1920px } from "./swiperHelpers.js";
+import Swiper from "./swiper-bundle.8.4.5.esm.browser.min.js";
+
+const swiperComments = new Swiper(".comments_slider_container", {
+
+	navigation: {
+		nextEl: '.swiper_button_next_comments',
+		prevEl: '.swiper_button_prev_comments',
+	},
+	scrollbar: {
+		el: ".swiper_scrollbar_2",
+		draggable: true,
+	},
+	slidesPerView: 1,
+	slidesPerGroup: 1,
+	spaceBetween: 10,
+	breakpoints: {
+		480: {
+			slidesPerView: 2,
+		},
+		768: {
+			slidesPerView: 2,
+		},
+		1000: {
+			slidesPerView: 3,
+			allowTouchMove: false,
+		},
+		1920: {
+			slidesPerView: 4,
+			spaceBetween: 15,
+			allowTouchMove: false,
+		}
+	},
+	speed: 1000,
+	focufocusableElements: "button",
+	on: {
+		slideChange: setSlidesState,
+	}
+});
+
+setSlidesState(swiperComments);
+switchSlidesStateHandlerOn1920px(swiperComments);
+
+const slidesOriginal = swiperComments.slides;
```

Comment:
you can add to `{ button, comment }` one more property called, for example `rootElem`